corehq/apps/smsforms/models.py

```diff
@@ -410,12 +408,15 @@ def _release_running_session_info_for_channel(cls, running_session_info, channel
 
     @staticmethod
     def _channel_affinity_cache_key(channel):
-        return f'XFormsSessionSynchronization.value.{channel.backend_id}/{channel.phone_number}'
```

Comment:
will this have any impact on existing sessions/claims?

Reply:
It shouldn't affect current, working sessions. If a session was able to be claimed and was running as expected this should match that existing key. @snopoke correct me if I'm wrong.